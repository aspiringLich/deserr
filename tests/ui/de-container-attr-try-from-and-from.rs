use deserr::Deserr;

#[derive(Deserr)]
#[deserr(try_from(String) = String::parse -> usize)]
#[deserr(from(String) = usize::FromStr)]
struct UnitStruct {
    hello: usize,
}

fn main() {}
