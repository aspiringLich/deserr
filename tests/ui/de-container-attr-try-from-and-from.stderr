error: The `from` and `try_from` attributes can't be used together.
 --> tests/ui/de-container-attr-try-from-and-from.rs:4:10
  |
4 | #[deserr(try_from(String) = String::parse -> usize)]
  |          ^^^^^^^^
